rpmio/expression.c

```diff
@@ -15,6 +15,7 @@
 
 #include <rpm/rpmlog.h>
 #include <rpm/rpmmacro.h>
+#include <rpm/rpmver.h>
```

Comment:
This change breaks the build for me here:

```
libtool: compile:  gcc -DHAVE_CONFIG_H -I.. -I.. -I../include/ -I../misc -DRPMCONFIGDIR=\"/usr/lib/rpm\" -DLOCALSTATEDIR=\"/var\" -I../luaext/ -D_REENTRANT -Wall -Wpointer-arith -Wmissing-prototypes -Wstrict-prototypes -fno-strict-aliasing -fstack-protector -Wempty-body -g -O2 -MT expression.lo -MD -MP -MF .deps/expression.Tpo -c expression.c  -fPIC -DPIC -o .libs/expression.o
expression.c:18:10: fatal error: rpm/rpmver.h: No such file or directory
   18 | #include <rpm/rpmver.h>
      |          ^~~~~~~~~~~~~~
compilation terminated.
```

Should it be "rpmio/rpmver.h" ?

Reply:
Nevermind, doing a git reset --hard, cleared some local changes to preinstall.am, which fixed it. Sorry for the noise.

Reply:
Yup, preinstall.am sometimes needs a little kick (aka rm -f) when public headers change.